styled-switch-view/src/main/java/co/sodalabs/view/StyledSwitchView.kt

```diff
@@ -191,6 +185,19 @@ class StyledSwitchView : ToggleableView {
         initProperties(attrs)
     }
 
+    override fun setOnOff(value: Boolean, userTriggered: Boolean) {
+        if (value != isActualOn) {
+            isActualOn = value
+            if (thumbAnimator?.isStarted == false && thumbAnimator?.isStarted == false) {
+                setupThumbBounds(thumbBounds, thumbOnCenterX, thumbOffCenterX, thumbRadius, value)
+            }
+
+            onToggledListener?.invoke(this, value, userTriggered)
```

Comment:
So, if we just pass the value directly to the listener, wouldn't the caller always know what the value of `userTriggered` is? In that case, what is the reason for passing it through at all?